packages/react-components/src/components/SendBox.tsx

```diff
@@ -217,12 +242,36 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   /* @conditional-compile-remove(file-sharing) */
-  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+  const uploadInProgressErrorBar = useMemo(() => {
+    const renderError = !!(
+      props.activeFileUploads?.length &&
+      !props.activeFileUploads
+        .filter((fileUpload) => !fileUpload.errorMessage)
+        .every((fileUpload) => fileUpload.isUploaded)
+    );
+    return renderError && <SendBoxErrorBar message={strings.fileUploadsPendingError} />;
+  }, [props.activeFileUploads, strings.fileUploadsPendingError]);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const fileUploadErrorBar = useMemo(() => {
     const fileUploads: ActiveFileUpload[] = props.activeFileUploads || [];
     const latestError = fileUploads.filter((fileUpload) => fileUpload.errorMessage).pop();
-    return <SendBoxErrorBar message={latestError?.errorMessage} timeout={10 * 1000} />;
+    return latestError && <SendBoxErrorBar message={latestError.errorMessage} timeoutDelay={10 * 1000} />;
   }, [props.activeFileUploads]);
 
+  /* @conditional-compile-remove(file-sharing) */
+  const renderSendBoxError = () => {
+    if (showFileUploadsPendingError) {
```

Comment:
Aha, I kinda get why this logic is so hard to write - we only want to show 1 error bar above sendbox so we put a condition to choose "this" and "that"
I am thinking of maybe it is not bad to show multiple error bar - pending error message won't eliminate those file upload messages - cause they are totally different messages and one should not override another - if we could show them all togehter instead of just one - things will be easier (But maybe we should ask AlexP for his suggestion to see which is the right experience)

Reply:
@pereiralex thoughts?

Reply:
Spoke to AlexP, we want to replace errors and only show 1 at a time.